src/file-uploader/file-uploader.component.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { FormsModule } from "@angular/forms";
+import { TestBed } from "@angular/core/testing";
+import { Component } from "@angular/core";
+import { ButtonModule } from "../button/button.module";
+import { LoadingModule } from "../loading/loading.module";
+import { FileUploader } from "./file-uploader.component";
+import { FileComponent } from "./file.component";
+import { CommonModule } from "@angular/common";
+import { CheckmarkFilled16Module } from "@carbon/icons-angular/lib/checkmark--filled/16";
+import { Close16Module } from "@carbon/icons-angular/lib/close/16";
+import { By } from "@angular/platform-browser";
+import { FileItem } from "./file-item.interface";
+
+@Component({
+	template: `
+		<ibm-file-uploader
+			title="title"
+			description="description"
+			buttonText="buttonText"
+			accept=".txt"
+			multiple="multiple"
```

Comment:
This is a boolean input ... maybe `true` would be better?

```suggestion
			multiple="true"
```